bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue

```diff
@@ -32,21 +32,20 @@
           <f7-preloader />
           <div>Loading...</div>
         </f7-block>
-        <!-- <f7-block class="block-narrow" v-if="ready && !previewMode">
-          <page-settings :page="page" :createMode="createMode" />
-        </f7-block> -->
 
-        <f7-block class="block-narrow" style="padding-bottom: 8rem" v-else-if="ready && modelReady && !previewMode">
-          <f7-col>
-            <f7-block-title>Page Configuration</f7-block-title>
-            <config-sheet
-              :parameterGroups="pageWidgetDefinition.props.parameterGroups || []"
-              :parameters="pageWidgetDefinition.props.parameters || []"
-              :configuration="page.config"
-              @updated="dirty = true" />
-          </f7-col>
+        <div v-else-if="!previewMode">
```

Comment:
I was just wondering why you removed the 2 other conditions (`ready && modelReady`).
Oh well I suppose it's fine...